screenpy/narration/adapters/__init__.py

```diff
@@ -3,11 +3,12 @@
 """
 
 
-from .stdout_adapter import StdOutAdapter
+from .stdout_adapter import StdOutAdapter, StdOutManager
 
 LoggingAdapter = StdOutAdapter
 
 __all__ = [
     "LoggingAdapter",
     "StdOutAdapter",
+    "StdOutManager",
```

Comment:
This is the only one that i hesitate on, i think. `StdOutManager` looks so similar to `StdOutAdapter` and these two are paired in a way that unless you're doin' something extra fancy (like a certain @bandophahita i know) you probably won't even need to know that `StdOutManager` even exists.

On the other hand, if we do want it front-and-center, i think we should write better documentation on how to use it.